lib/lib-table-cluster/src/test/java/org/sagebionetworks/table/cluster/TableInfoTest.java

```diff
@@ -1,67 +1,256 @@
 		TableNameCorrelation tableNameCorrelation = new TableQueryParser("syn123 t").tableNameCorrelation();
 		// call under test
-		TableInfo info = new TableInfo(tableNameCorrelation);
+		TableInfo info = new TableInfo(tableNameCorrelation, schema);
 		assertEquals("syn123", info.getOriginalTableName());
 		assertEquals(IdAndVersion.parse("syn123"), info.getTableIdAndVersion());
 		assertEquals("T123", info.getTranslatedTableName());
 		assertEquals("t", info.getTableAlias().orElse(null));
+		assertEquals(schema, info.getTableSchema());
 	}
-	
+
 	@Test
 	public void tesConstructorWithVersionNoAlias() throws ParseException {
 		TableNameCorrelation tableNameCorrelation = new TableQueryParser("syn123.4").tableNameCorrelation();
 		// call under test
-		TableInfo info = new TableInfo(tableNameCorrelation);
+		TableInfo info = new TableInfo(tableNameCorrelation, schema);
 		assertEquals("syn123.4", info.getOriginalTableName());
 		assertEquals(IdAndVersion.parse("syn123.4"), info.getTableIdAndVersion());
 		assertEquals("T123_4", info.getTranslatedTableName());
 		assertEquals(null, info.getTableAlias().orElse(null));
+		assertEquals(schema, info.getTableSchema());
 	}
-	
+
 	@Test
 	public void tesConstructorWithVersionWithAlias() throws ParseException {
-		TableNameCorrelation tableNameCorrelation = new TableQueryParser("syn123.4 as \"has space\"").tableNameCorrelation();
+		TableNameCorrelation tableNameCorrelation = new TableQueryParser("syn123.4 as t")
+				.tableNameCorrelation();
 		// call under test
-		TableInfo info = new TableInfo(tableNameCorrelation);
+		TableInfo info = new TableInfo(tableNameCorrelation, schema);
 		assertEquals("syn123.4", info.getOriginalTableName());
 		assertEquals(IdAndVersion.parse("syn123.4"), info.getTableIdAndVersion());
 		assertEquals("T123_4", info.getTranslatedTableName());
-		assertEquals("\"has space\"", info.getTableAlias().orElse(null));
+		assertEquals("t", info.getTableAlias().orElse(null));
+		assertEquals(schema, info.getTableSchema());
 	}
-	
+
 	@Test
 	public void testConstructorWithNull() {
 		TableNameCorrelation tableNameCorrelation = null;
-		String message = assertThrows(IllegalArgumentException.class, ()->{
-			new TableInfo(tableNameCorrelation);
+		String message = assertThrows(IllegalArgumentException.class, () -> {
+			new TableInfo(tableNameCorrelation, schema);
 		}).getMessage();
 		assertEquals("TableNameCorrelation is required.", message);
 	}
+	
+	@Test
+	public void testLookupColumnReferenceWithoutLHS() throws ParseException {
+		TableNameCorrelation tableNameCorrelation = new TableQueryParser("syn123")
+				.tableNameCorrelation();
+		ColumnReference reference = new TableQueryParser("foo").columnReference();
+		TableInfo info = new TableInfo(tableNameCorrelation, schema);
+		// call under test
+		Optional<ColumnTranslationReference> transRef = info.lookupColumnReference(reference);
+		assertNotNull(transRef);
```

Comment:
assertEquals(value, transRef.get())? or alternatively: assertEquals(Optional.of(value), transRef))